.github/lighthouse/lighthouse-config.js

```diff
@@ -2,12 +2,11 @@ export default {
     extends: 'lighthouse:default',
     settings: {
         onlyCategories: [
-            // Excluded categories do not currently pass
-            // "accessibility",
-            // "best-practices",
+            'accessibility',
```

Comment:
Turning on the categories here does not seem to turn on the audits. This way we can report their scores to the PR.